### PR TITLE
add the implementation files

### PR DESCRIPTION
**Pull request by eversrx19**
1. pytorch model
2. sign up/in
3. upload feature
4. test image file

### CODE REVIEW

File: main/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# '__init__.py'는 폴더의 파이썬 파일 중 가장 먼저 실행 
+# app이라는 flask 객체를 만들어 여기서 blueprint 등록해주는 작업 수행(register_blueprint)
+from flask import Flask
+from flask_cors import CORS
+from flask import Blueprint, jsonify, request
+import torch
+#from . import signup
+#from . import login
```
Comment (eversrx19) [inline main/__init__.py:8]: please delete this comment

File: main/result.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+device = "cpu"
+# device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+model = models.resnet34(pretrained=True)
+num_ftrs = model.fc.in_features
+model.fc = nn.Linear(num_ftrs, 3)
+checkpoint = torch.load('main/static/model/path.pth', map_location=torch.device('cpu'))
+model.load_state_dict(checkpoint)
+# model = TheModelClass(*args, **kwargs)
+# model.load_state_dict(torch.load('main/static/model/model.pth', map_location=torch.device('cpu')))
+# model = torch.load('main/static/model/model.pth', map_location=torch.device('cpu'))
```
Comment (eversrx19) [inline main/result.py:31]: same thing

File: main/index.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+def upload():
+    print(request.files)
+    file = request.files['file_give']             # 파일 받기
+    extension = file.filename.split('.')[-1]      # 확장자
+    today = datetime.datetime.now()                        # 현재 날짜, 시간
+    mytime = today.strftime('%Y-%m-%d-%H-%M-%S')  #날짜 시간을 원하는 형태로 변환
+    filename = f'{mytime}.{extension}'  # 파일 이름
+    save_to = f'/static/img/{filename}' # 파일 경로
+    test = os.path.abspath(__file__)    # ?
+    parent_path = Path(test).parent     # ?
```
Comment (eversrx19) [inline main/index.py:20]: ?

Comment (J1NU2) [reply main/index.py:20]: ?

**Comment by eversrx19** [comment]
LGTM
looks good to me

**Pull request merged**